index.js

```diff
@@ -112,6 +121,7 @@ var run = function(options) {
     }
 
     if (options.debug) console.log("All done.");
+    process.exit(0);
```

Comment:
That was there because knex was not closing it's connection to the database, keeping the process running. I just amended the initial commit to include changes that close the db connection once the data has been inserted. This line is no longer present in that commit.